assignment/scripts/loops-practice.js

```diff
@@ -13,35 +12,66 @@ for (let i=0; i<4; i++) {
 // 1.a. TODO: Write a for loop to console.log the numbers from 0 to 5 
 //   - Which part of the example loop do you need to change to do this?
 console.log('count from 0 to 5');
+for (let i=0; i<6; i++) {
```

Comment:
FWIW, it is more standard to see spaces around these operators (but not `++`).

```suggestion
for (let i = 0; i < 6; i++) {
```